docs/setup-android.md

```diff
@@ -22,16 +22,18 @@ In order to integrate CodePush into your Android project, please perform the fol
 1. In your `android/settings.gradle` file, make the following additions:
 
     ```gradle
-    include ':app', ':react-native-code-push'
+    include ':app', ':react-native-code-push'. // Append ':react-native-code-push' to :app
     project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
     ```
     
-2. In your `android/app/build.gradle` file, add the `codepush.gradle` file as an additional build task definition underneath `react.gradle`:
+2. In your `android/app/build.gradle` file, add the `codepush.gradle` file as an additional build task definition underneath `react.gradle` or `native_modules.gradle`(in case of RN [build issue](https://github.com/facebook/react-native/issues/25631)):
```

Comment:
Thank you for your contribution, but in fact, I think that this does not apply to setting up CodePush, but is related to setting up the react-native app, and should not appear in our documentation. In my opinion, we should remove this so as not to confuse people who do not use `native_modules.gradle`. Could update this PR and remove this change. If you have a different opinion, let me know and we will discuss it.